metrics/plugins/json/README.md

```diff
@@ -0,0 +1,65 @@
+## Plain
+
+Plugin to collect metrics and encode to json string.
+
+### API
+
+Import json plugin:
+
+```lua
+local json_metrics = require('metrics.plugins.plain')
+```
+
+#### `json.export()`
+Returns:
+```lua
+string:
+{
+    "<name_obs>{<key>=\"value\"}":{
+        "timestamp":<number>,
+        "value":<value>
+    }
+}
+```
+**Important** - values can be **+-math.huge**, **math.huge * 0**
+In such cases, the value will be represented by the string, in other cases the number.
+
+Example:
+```lua
+string:
+{
+    "name{label=\"-math.huge\"}":{
```

Comment:
Consumer of this json will be mad at you :)
You force him to parse this key in order to retrieve `name` and `label`.

I propose an array of flat dictionaries:
```json
{
  "name": "my_metric_name",
  "label": "-inf",
  "timestamp": 1558699055876857,
  "value": "-inf",
}
```